hugegraph-pd/hg-pd-client/pom.xml

```diff
@@ -34,7 +34,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.20</version>
+            <version>1.18.30</version>
```

Comment:
Can we remove the lombok dependency:
since we don't have a large places to generate/replace code, and in the few places where it's necessary, I would rather write some code manually to maintain a consistent style.